projects/module-2/whats-cookin-part-one.md

```diff
@@ -0,0 +1,277 @@
+---
+title: What's Cookin - Part one!
+tags: javascript, mocha, testing, fetch
+---
+## Goals and Objectives
+
+- **Use object and array prototype methods to perform data manipulation**
+- Create a user interface that is easy to use and clearly displays information.
+- Write modular, reusable code that follows SRP (Single Responsibility Principle)
```

Comment:
I think we can remove "modular" and just keep "reusable" since we'd love for them to write functions that can be reused if possible (DRY), leveraging args to make them dynamic..

Reply:
As to your question about if other FP principles should be specifically called out in the rubric, maybe this bullet point could also include something about functions being DRY and making an effort toward purity (or I think it's fine if this language only lives in the rubric, too)

Reply:
Replaced "modular" with "DRY".  

Added note about purity to javascript competency examples in rubric.